LibCarla/source/carla/Exception.cpp

```diff
@@ -14,14 +14,14 @@
 #ifdef BOOST_NO_EXCEPTIONS
 
 namespace boost {
-
+  
```

Comment:
remove empty space pls

Reply:
fixed by calling Util/Formatting/codeformat.py and placing .clang-format to LibCarla folder